packages/web/src/app/orbs/page.tsx

```diff
@@ -0,0 +1,101 @@
+interface PowerInfo {
+  name: string;
+  category: string;
+  memberOnly?: boolean;
+  slug: string;
+}
+
+const powers: PowerInfo[] = [
+  {
+    name: "Multiply",
+    category: "Basic",
+    slug: "multiply",
+  },
+  {
+    name: "Cancel Multiply",
+    category: "Basic",
+    slug: "cancel-multiply",
+  },
+  {
+    name: "Orbic Rehash",
+    category: "Basic",
+    slug: "orbic-rehash",
+  },
+  {
+    name: "Grow Quadradius",
+    category: "Enhancement",
+    slug: "grow-quadradius",
+  },
+  {
+    name: "Bombs",
+    category: "Combat",
+    slug: "bombs",
+  },
+  {
+    name: "Smart Bombs",
+    category: "Combat",
+    slug: "smart-bombs",
+  },
+  {
+    name: "Invisible",
+    category: "Movement",
+    memberOnly: true,
+    slug: "invisible",
+  },
+  {
+    name: "Recruit Radial",
```

Comment:
Not necessarily required for now but I think that since this is an upgrade from older doc then we could just explain how the powerups work and omit the row/column/radial for each of them and just explain what these mean separately